python/tests/rtl/test_api_methods.py

```diff
@@ -0,0 +1,119 @@
+import datetime
+import pytest
+from typing import MutableMapping, Optional
+
+from looker_sdk.rtl import api_methods as am
+from looker_sdk.rtl import requests_transport as rtp
+from looker_sdk.rtl import serialize as sr
+from looker_sdk.rtl import transport as tp
+from looker_sdk.rtl import user_session as us
+from looker_sdk.rtl import model
+from looker_sdk.sdk import models as ml
+
+
+@pytest.fixture(scope="module")
+def api_methods() -> am.APIMethods:
+    return am.APIMethods(
+        us.UserSession, sr.deserialize, sr.serialize, rtp.RequestsTransport
+    )
+
+
+@pytest.mark.parametrize(
+    "input, expected",
+    [
+        ({"a": None}, {}),
+        ({"a": True}, {"a": "true"}),
+        ({"a": "text"}, {"a": "text"}),
+        ({"a": 1}, {"a": "1"}),
+        ({"a": [1, 2, 3]}, {"a": "[1, 2, 3]"}),
+        ({"a": ["x", "xy", "xyz"]}, {"a": '["x", "xy", "xyz"]'}),
+        ({"a": datetime.datetime(2019, 8, 14, 8, 4, 2)}, {"a": "2019-08-14T08:04Z"}),
+    ],
+)
+def test_convert_query_params(
+    api_methods: am.APIMethods,
+    input: am.TQueryParams,
+    expected: MutableMapping[str, str],
+):
+    actual = api_methods._convert_query_params(input)
+    assert actual == expected
+
+
+@pytest.mark.parametrize(
+    "input, expected",
+    [
+        ("some body text", b"some body text"),
+        ("", b""),
+        ({}, None),
+        (ml.WriteApiSession(workspace_id="dev"), b'{"workspace_id": "dev"}'),
+        (
+            ml.ApiVersion(
+                looker_release_version="6.18",
+                current_version=ml.ApiVersionElement(
+                    full_version="6.18.4", status="fully functional"
+                ),
+            ),
+            b'{"looker_release_version": "6.18", "current_version": {"full_version": "6.18.4", "status": "fully functional"}}',  # noqa: B950
+        ),
+        (
+            ml.ApiVersion(
+                looker_release_version="6.18",
+                current_version=ml.ApiVersionElement(
+                    full_version="6.18.4", status="fully functional"
+                ),
+                supported_versions=model.EXPLICIT_NULL,
```

Comment:
the intention is for the end user to use
```suggestion
                supported_versions=ml.EXPLICIT_NULL,
```
so they won't have to import any rtl code ditto for the other places and then you can remove the model import above